openQA.spec

```diff
@@ -410,6 +410,13 @@ fi
 %{_tmpfilesdir}/openqa-webui.conf
 # web libs
 %dir %{_datadir}/openqa
+%{_datadir}/openqa/lib/DBIx/
+%{_datadir}/openqa/lib/LiveHandler.pm
```

Comment:
That should probably be `%{_datadir}/openqa/lib/OpenQA/LiveHandler.pm`.

Reply:
thx, fixed. Also for the intermediate commit about ServerSideDataTable